components/markets/MarketHeader.tsx

```diff
@@ -367,6 +378,20 @@ const MarketHeader: FC<{
             dateStyle: "medium",
           }).format(ends)}
         </HeaderStat>
+        {(market.status === "Active" || market.status === "Closed") && (
```

Comment:
for consistency with the other stats; you could do something like this for the resolution date passes: `<HeaderStat label={hasDatePassed(resolutionDateEstimate) ? "Resolved" : "Resolves"}>`

Reply:
Once the market resolves the same information is shown here so I don't think we need to display it
<img width="345" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/da13fd0d-1177-424a-b679-9f204b951a91">